index.html

```diff
@@ -207,7 +207,27 @@ <h3>
               "type": "image/jpeg"
             }]
           }
-      </pre>
+        </pre>
+        <p>
+          The following shows a <a>manifest</a> that prefers the <code>
+          minimal-ui</code> {{DisplayModeType}}
+          to <code>standalone</code>, but wants to support both.
```

Comment:
s/wants to support both/if `minimal-ui` isn't supported, fall back to `standalone` as opposed to `browser`.